tests/nfs3-tcp-frames/test.yaml

```diff
@@ -0,0 +1,50 @@
+requires:
+  min-version: 7.0
+
+args:
+- -k none
+- --set stream.midstream=true
+- --set app-layer.protocols.nfs.enabled=yes
```

Comment:
nit: considering that we have nfs enabled in the suricata.yaml file, I think this line isn't necessary ;)